app/views/static_pages/index.html.erb

```diff
@@ -1,3 +1,13 @@
-<h1 class="col-10 offset-1">
-	Welcome
-</h1>
+
+
+<div class="booyah-box col-4">
+<h1>Join a Game</h1>
+
+<% @games.available.each do |game| %>
+    <% if game.white_player %>
+      <%= game.name + "'s game" %>
+    <% elsif game.black_player %>
```

Comment:
If you try to create a game with no white player, validation fails and game is not created. Therefore, this won't every get reached.

Reply:
Ok well thats another issue entirely, we should be able to fix that in the controller method but no change needs to made to this code. We should be able to create a game as a white or black player.

Reply:
I was thinking I read that only the white player could start a game. I think I was wrong about that

This looks good